build/dependencies.props

```diff
@@ -0,0 +1,46 @@
+﻿<Project>
```

Comment:
I don't know how these numbers are determined or how they get updated, but I trust that's automated somehow as part of your replacement for lineups.

Reply:
There are two scenarios to this. 

1. Automatically flowing new versions when building aspnet/Templating in Universe. Universe generates a file with new versions and sets DotNetPackageVersionPropsPath. This will override versions set in the repo source code.

https://github.com/aspnet/templating/blob/c240aaa1aef89a3b5a6f765a9bb2229f10b1cbec/build/dependencies.props#L45

2. Updating the repo source code. We haven't automated this yet, but we plan to. Before we make these changes in dev, we'll make some command-line tools to make updating to latest easier. We'll also add some check to ensure the right variable names are used, floating package versions are not, and that that all PackageReference's use a variable.